safe_check.py

```diff
@@ -31,7 +31,7 @@
     try:
       e.lineno = e.lineno - \
           len(encoding_header.ENCODING_DECLARATION.splitlines())
-    except AttributeError:
+    except (AttributeError, TypeError):
       # Some exceptions may not have line numbers.  If so, ignore
```

Comment:
Should we update the comment as well? e.g.:
`# Ignore exceptions without line number attribute or wrong line number type`

This is a non-blocker, though. Feel free to merge!